### PR TITLE
Backporting for 2.375.3

### PR DESCRIPTION
**Pull request by NotMyFault**
```
Latest core version: jenkins-2.387

Fixed
-----

JENKINS-70209           Minor                   2.385
        Letter spacing should not be negative (regression in 2.340 + 2.350)
        regression
        https://issues.jenkins.io/browse/JENKINS-70209

JENKINS-70301           Minor                   2.386
        Resolve implied dependencies on WMI Windows Agent plugin
        https://issues.jenkins.io/browse/JENKINS-70301

JENKINS-70169           Minor                   2.385
        Missing breadcrumb in Console log view
        regression
        https://issues.jenkins.io/browse/JENKINS-70169

JENKINS-70240           Minor                   2.383
        Non-HTTP based UpdateCenter URLs cause exception in plugin manager
        regression
        https://issues.jenkins.io/browse/JENKINS-70240
```

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7590"></a>

### CODE REVIEW

File: core/src/main/java/hudson/slaves/ChannelPinger.java [hunk 1]
```diff
@@ -195,6 +195,14 @@ protected void onDead(Throwable cause) {
                         LOGGER.log(Level.FINE, "Ping failed after the channel " + channel.getName() + " is already partially closed.", cause);
                     } else {
                         LOGGER.log(Level.INFO, "Ping failed. Terminating the channel " + channel.getName() + ".", cause);
+                        if (computer == null) {
+                            // Disconnect from agent side.
```
Comment (jglick) [inline core/src/main/java/hudson/slaves/ChannelPinger.java:199]: From https://github.com/jenkinsci/jenkins/pull/7580. Normally we wait for a fix to be in weeklies for a little while before proposing a backport, right?

Comment (NotMyFault) [reply core/src/main/java/hudson/slaves/ChannelPinger.java:199]: The RC is scheduled for next week's Wednesday, don't you think 8 days is enough to test this in a weekly?

Comment (jglick) [reply core/src/main/java/hudson/slaves/ChannelPinger.java:199]: No particular opinion (and as author of the change I would be biased anyway). I recall there being some policy we follow generally, and this does not seem to be an exceptional case—something that has been broken for years, not a recent regression.

Comment (NotMyFault) [reply core/src/main/java/hudson/slaves/ChannelPinger.java]: > something that has been broken for years, not a recent regression.

I see, I missed the 'causes' part on the Jira issue. 2016-era isn't recent, indeed. I dropped the commit.

**Review by MarkEWaite — APPROVED** [review]

**Review by basil — COMMENTED** [review]
At the time of this writing [JENKINS-70240](https://issues.jenkins.io/browse/JENKINS-70240) remains rejected, for a reason I believe is invalid as explained in [JENKINS-70240 (comment)](https://issues.jenkins.io/browse/JENKINS-70240?focusedCommentId=434217&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-434217).

**Review by basil — APPROVED** [review]
Between this PR and https://github.com/jenkinsci/jenkins/pull/7597, the content of 2.375.3 looks good to me (though I see no reason to retain the `2.375.3-rejected` label on [JENKINS-70240](https://issues.jenkins.io/browse/JENKINS-70240)).

**Pull request merged**